src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -60,54 +60,8 @@ internal static bool TryGetDataContractFromGeneratedAssembly(Type type, out Data
 
         internal static DataContract GetDataContractFromGeneratedAssembly(Type type)
         {
-#if uapaot
-            if (DataContractSerializer.Option == SerializationOption.ReflectionOnly)
-            {
-                return null;
-            }
-
-            type = GetDataContractAdapterTypeForGeneratedAssembly(type);
-            DataContract dataContract = GetGeneratedDataContract(type);
-            if (dataContract == null)
-            {
-                if (type.IsInterface && !CollectionDataContract.IsCollectionInterface(type))
-                {
-                    type = Globals.TypeOfObject;
-                    dataContract = GetGeneratedDataContract(type);
-                }
-                if (dataContract == null)
-                {
-                    if (DataContractSerializer.Option == SerializationOption.CodeGenOnly)
-                    {
-                        throw new InvalidDataContractException(SR.Format(SR.SerializationCodeIsMissingForType, type));
-                    }
-                }
-            }
-
-            if (dataContract is InvalidDataContract && DataContractSerializer.Option == SerializationOption.ReflectionAsBackup)
-            {
-                return null;
-            }
-
-            return dataContract;
-#else
             return null;
```

Comment:
Seems like there could subsequently be additional cleanup in this project, as there are now several empty methods, setters, etc.